packages/core/src/repository/data-source/sql/raw-sql.data-source.ts

```diff
@@ -71,6 +72,8 @@ export class RawSQLDataSource implements GetDataSource<RawSQLData>, PutDataSourc
         this.tableColumns = tableColumns.concat(columns);
     }
 
+    public softDeleteEnabled = false;
+
```

Comment:
I though you were going to pass this on the constructor or derive it from `deleteAtColumn`. Make `deleteAtColumn` optional without a default value and if it's defined the assume that it's soft delete?

Reply:
That's correct. I just rushed to code what I had before. Good catch.

Reply:
moved to the constructor.